scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -527,20 +527,37 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm solve case</h5>
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">
-        Are you sure? Marking a case as Solved will queue it to delete analysis files
-        ahead of schedule, and disable the alignment view.
-        You will then have to request a FULL rerun to continue evaluating e.g. research variants.
-        <br><br>Marking Solved directly does not store the Causative variant.
-        Only use it for cases where the causation is not readily markable, such as UPD or SMN.
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-        <form method="POST"
+      <form method="POST"
           action="{{ url_for('cases.status', institute_id=institute._id, case_name=case.display_name) }}">
-          <button type="submit" class="btn btn-primary" value="solved" name="status">Solve</button>
-        </form>
+        <div class="modal-body">
+          Are you sure? Marking a case as Solved will queue it to delete analysis files
+          ahead of schedule, and disable the alignment view.
+          You will then have to request a FULL rerun to continue evaluating e.g. research variants.
+          <br><br>Marking Solved directly does not store the Causative variant.
+          Only use it for cases where the causation is not readily markable, such as UPD or SMN.
+          <br><br>Tag case status below.
+          Case status tags are used for statistics and follow up. Fill them as accurately as possible.
+         <div class="row">
```

Comment:
Perhaps add a it of spacing from top? A broader select would also be more readable in the case of several options selected

<img width="398" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/e6159173-bc43-4cb2-8faf-2093afc0ff0b">

Reply:
Agreed! I've added some space. The widths are a bit tricky; one can't let them run dynamically to max size, as they could potentially become very wide. On the other hand, making them statically wide is silly as there would only very rarely be more than one checked at the time. Do you know of any good middle ground, like a dynamic-with-a-sensible-max limit for selectpicker?

Reply:
Perhaps if you assign it the size you want to have in columns? Like col-4 ? But I don't know if it will work

Reply:
Toggling the `data-width` explicitly for this one seemed to help, even if the page is supposed to have a general 100% for all selectpickers.
![Screenshot 2024-02-19 at 10 34 28](https://github.com/Clinical-Genomics/scout/assets/758570/cf4b293e-6cc7-4cb5-8d7f-2e40f46aff56)